packages/rspack/src/generators/configuration/configuration.ts

```diff
@@ -76,8 +76,11 @@ function addBuildTarget(tree: Tree, options: ConfigurationSchema) {
         defaultConfiguration: 'production',
         options: buildOptions,
         configurations: {
-          development: {},
+          development: {
+            mode: 'development',
```

Comment:
This should be within `buildOptions` I think.

Reply:
In `vite` we have it there, too. https://github.com/nrwl/nx/blob/master/packages/vite/src/utils/generator-utils.ts#L208

So, I wanted to keep it consistent!